src/lib/bootloader/grub2pwd.rb

```diff
@@ -120,28 +121,23 @@ def disable
       return unless used_on_target?
 
       # operate on target as we have to remove password during installation from target grub2
-      Yast::SCR.Execute(Yast::Path.new(".target.bash"), "rm '#{PWD_ENCRYPTION_FILE}'")
+      Yast::SCR.Execute(Yast::Path.new(".target.bash"), "rm '#{PWD_ENCRYPTION_FILE.shellescape}'")
     end
 
     def encrypt(password)
-      Yast.import "String"
+      result = Yast::Execute.locally("/usr/bin/grub2-mkpasswd-pbkdf2",
+        env:    { "LANG" => "C" },
+        stdin:  "#{password}\n#{password}\n",
```

Comment:
Please add this example to https://yastgithubio.readthedocs.io/en/latest/security-tips/#passing-sensitive-data-to-external-tools, it would be nice to have real YaST example.

Reply:
ok, will prepare it tomorrow

Reply:
@lslezak please review it at https://github.com/yast/yast.github.io/pull/171